src/lib.rs

```diff
@@ -105,7 +105,12 @@ use {
 };
 
 use sync::atomic::{AtomicPtr, AtomicUsize, Ordering};
-use sync::{Arc, WithMut};
+use sync::Arc;
+
+#[cfg(not(loom))]
+// WitMut is not needed under loom because loom::sync::atomic::AtomicPtr includes an intrinsic
+// with_mut that has the same interface as this crate's sync::WithMut::with_mut
```

Comment:
This comment is not necessary; the `WithMut` trait was added entirely in preparation for `loom`.